src/main/java/edu/usf/cutr/gtfsrtvalidator/api/resource/GtfsRtFeed.java

```diff
@@ -152,8 +162,13 @@ public Response startMonitor(
         session.save(sessionModel);
         GTFSDB.commitAndCloseSession(session);
 
-        //Extract the Url and gtfsId to start the background process
-        startBackgroundTask(gtfsRtFeed, updateInterval);
+        if(feedSession == null) {
```

Comment:
@barbeau ok sounds good. I removed those changes and updated the pull request.
Yeah I would also be surprised then. I'm more than happy to help but I would need to know what is wrong with the screenshot I posed in the summary field so I can look more into it. It's not obvious to me where the mix of feed appears but I guess I'm confused by what it shows. 
That being said, I'm off from Friday for a week and a half...By that time your team would have probably figured out the fix!